OWNERS_ALIASES

```diff
@@ -0,0 +1,19 @@
+# See the OWNERS_ALIASES docs: https://git.k8s.io/community/contributors/guide/owners.md#owners_aliases
+
+aliases:
+  ci-approvers:
+    - Sandeepyadav93
+    - viroel
+  network-approvers:
+    - beagles
+    - gthiemonge
+    - johnsom
+    - karelyatin
+    - slawqo
+    - weinimo
+  openstack-approvers:
+    - abays
+    - dprince
+    - gibizer
+    - olliewalsh
+    - stuggi
```

Comment:
~~~
fix end of files.........................................................Failed
- hook id: end-of-file-fixer
- exit code: 1
- files were modified by this hook
Fixing OWNERS_ALIASES
don't commit to branch..................................................Skipped
12 skipped lines...
     - gibizer
     - olliewalsh
-    - stuggi
\ No newline at end of file
+    - stuggi
~~~

Reply:
johnsom and gthiemonge will need to join openstack-k8s-operators org to be part of OWNERS file, you can ping on #openstack-k8s-operators channel and someone can invite them to this group.

Reply:
done already. thanks